src/Logger.php

```diff
@@ -314,11 +319,11 @@ private function dispatch(array $messages, bool $final): void
                     $target->collect($messages, $final);
                 } catch (Throwable $e) {
                     $target->disable();
-                    $targetErrors[] = [
+                    $targetErrors[] = new Message(
                         LogLevel::WARNING,
                         'Unable to send log via ' . get_class($target) . ': ' . get_class($e) . ': ' . $e->getMessage(),
```

Comment:
Here we likely should use key from `$this->getTargets()` since in `getTarget()` we assume that the target is named.